Dotnet/v32.0.0

ameetj85 opened the pull request:
Added starter samples for Asp.Net Core MVC.
Added and Updated interop samples for v1 and v2 adapter.

ameetj85 committed:
interop-example for v1

ameetj85 committed:
Added Windows Forms example of InterOp. Ported over from v1 and modified. NOT TESTED as yet.

ameetj85 committed:
Added Asp.Net Core MVC starter samples

ameetj85 committed:
Updated styling to match workspace starter UI.

johnman reviewed (changes requested):
Hi @ameetj85 can we have a root folder for web examples (so if we have more than one they don't start polluting the root of how-to)?

maybe 
how-to/web or how-to/server (as there is a difference between client side .NET WPF, WinForm, Console and server side based solutions asp.net etc).

Can you also merge the integrate-with-workspace updates (where the path changed).

johnman committed:
Merge branch 'main' into dotnet/v32.0.0

johnman committed:
Dev/john/update interop example (#26)

Applied the following changes:

- Removed the webview
- Replaced the webview with a logging window
- Added support for raising more intents (ViewNews, ViewInstrument)
- Added support for listening for intents: ViewNews, ViewInstrument and ViewContact.
- Added a better example of how to fire an intent.
- Added a FDC3 folder with intent and context types for use in the example.
- Updated nuget references to secure versions
- Added option to pass settings to the winform app via the command line.

gilesstevenson committed:
Various fixes to Interop sample, for contexts and intents.

gilesstevenson committed:
Enable the Interop Broker functionality

ameetj85 committed:
Moved asp.net examples under the server root folder. Updated readme.md to reflect this

ameetj85 commented:
@johnman moved asp.net under server. Also, the integrate-with-workspace app.config is updated with the new path to the manifest and uuid.
<add key="workspaceManifestUrl" value="https://built-on-openfin.github.io/workspace-starter/main/workspace-platform-starter/manifest.fin.json" />
<add key="workspaceUUID" value="workspace-platform-starter" />
